ui-speedspacechart/src/components/layers/SpeedLimitTagsLayer.tsx

```diff
@@ -1,21 +1,42 @@
-import React, { useEffect, useRef } from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 
 import type { Store, tooltipInfos } from '../../types/chartTypes';
 import Tooltip from '../common/Tooltip';
 import { LINEAR_LAYERS_HEIGHTS } from '../const';
 import { drawSpeedLimitTags, computeTooltip } from '../helpers/drawElements/speedLimitTags';
+import { createSvgBlobUrl, loadSvgImage } from '../utils';
 
 type SpeedLimitTagsLayerProps = {
   width: number;
   marginTop: number;
   store: Store;
 };
+type LoadedImages = {
+  questionImage: HTMLImageElement | null;
+  alertFillImage: HTMLImageElement | null;
+};
 const TOOLTIP_HEIGHT = 40;
 const MARGIN_ADJUSTMENT = 2;
 
 const SpeedLimitTagsLayer = ({ width, marginTop, store }: SpeedLimitTagsLayerProps) => {
   const canvas = useRef<HTMLCanvasElement>(null);
   const tooltip = useRef<tooltipInfos | null>();
+  const [images, setImages] = useState<LoadedImages>({ questionImage: null, alertFillImage: null });
+
+  useEffect(() => {
+    const fetchImages = async () => {
+      const questionSvg =
+        '<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 16 16"><path d="M0 8a8 8 0 1 1 16 0A8 8 0 0 1 0 8m8-6.5a6.5 6.5 0 1 0 0 13 6.5 6.5 0 0 0 0-13M6.92 6.085h.001a.75.75 0 1 1-1.342-.67c.169-.339.436-.701.849-.977C6.845 4.16 7.369 4 8 4a2.76 2.76 0 0 1 1.637.525c.503.377.863.965.863 1.725 0 .448-.115.83-.329 1.15-.205.307-.47.513-.692.662-.109.072-.22.138-.313.195l-.006.004a6 6 0 0 0-.26.16 1 1 0 0 0-.276.245.75.75 0 0 1-1.248-.832c.184-.264.42-.489.692-.661q.154-.1.313-.195l.007-.004c.1-.061.182-.11.258-.161a1 1 0 0 0 .277-.245C8.96 6.514 9 6.427 9 6.25a.61.61 0 0 0-.262-.525A1.27 1.27 0 0 0 8 5.5c-.369 0-.595.09-.74.187-.146.1-.263.238-.34.398M9 11a1 1 0 1 1-2 0 1 1 0 0 1 2 0"/></svg>';
```

Comment:
Orthogonal to this PR: I've been meaning to stop putting SVG data into strings and instead embed them with e.g. `@rollup/plugin-image`.

Reply:
I don't quite understand the implications of different image loading methods, but feel free to commit something if you think an improvement can be made in this area